spec/controllers/portal/client_logins_controller_spec.rb

```diff
@@ -209,6 +210,13 @@
             expect(response).to redirect_to portal_root_path
           end
 
+          it "updates the clients last_seen_at" do
```

Comment:
Can you change the use of Timecop so that you freeze to a particular time e.g. 

```Timecop.freeze(DateTime.new(2021, 8, 1, 12, 0, 0)) do
  expect(...).to eq(DateTime.new(2021, 8, 1, 12, 0, 0
  end```
  
As written, your test is not a very idiomatic usage of Timecop, since you aren't specifying _what_ time to freeze to.

Reply:
If you do that, you'll probably enjoy having a `fake_time` variable so you aren't repeating yourself.

Also FYI I edited the above comment like 10 times after saving; sorry if the previous 9 versions were just distractingly incorrect. I hope this version is adequately correct.

Reply:
https://frontdeveloper.pl/2020/03/how-we-migrated-from-timecop-to-built-in-rails-5-2-time-helpers/ is a good reference, and also it shows that we can get rid of Timecop entirely :P

Reply:
I'm OK with this!

Reply:
@tgrathwell @bytheway875 Do you have an opinion on `ActiveSupport::Testing::TimeHelpers` vs `Timecop`?

It seems that Rails 5.2 integrates something very similar to Timecop, and that life will be simpler if we use it.

If you think it's sensible to use `ActiveSupport::Testing::TimeHelpers`, then I propose we do this:

```
RSpec.configure do |config|
  ....
  config.include ActiveSupport::Testing::TimeHelpers
  ....
end
```

in `spec/rails_helper.rb`

Reply:
@caitbot For now, for this pull request, consider making the above change to `spec/rails_helper.rb` and removing the `include` from this file. Not a blocker.

Reply:
honestly, im An Old and haven't kept track of what has been added to rails to replace the things in my toolbox. if we're doing things that make external dependencies obsolete, im always cool w/ that.

However, if the code is valid and my Old way fits the pattern we'd been using before and we're two days away from a launch, i suggest we make a ticket to improve it across the board later and move forward with whatever fits the existing pattern we'd used before.